pkg/core/helper_test.go

```diff
@@ -210,6 +210,18 @@ func TestCreateBasicChain(t *testing.T) {
 	b.Header().EncodeBinary(buf.BinWriter)
 	t.Logf("header: %s", hex.EncodeToString(buf.Bytes()))
 
+	// Move some nep5 GAS to the same account to be able to deploy transactions
+	gasHash := bc.contracts.GAS.Hash
+	t.Logf("native GAS hash: %v", gasHash)
+	txMoveGas := newNEP5Transfer(gasHash, neoOwner, priv0ScriptHash, 10000000)
+	txMoveGas.Nonce = getNextNonce()
+	txMoveGas.ValidUntilBlock = validUntilBlock
+	txMoveGas.Sender = neoOwner
+	require.NoError(t, signTx(bc, txMoveGas))
+	b = bc.newBlock(txMoveGas)
```

Comment:
We could probably pack `txMoveGas` in the same transaction with `txMoveNeo`, it would simplify things a bit.